src/tests/json-strings/validationStrings.ts

```diff
@@ -904,7 +904,10 @@ export const documentInvalidPropertiesState = `{
           "Resource": "arn:aws:lambda:us-east-1:111111111111:function:FUNCTION_NAME",
           "Next": "ChoiceState",
           "SomethingInvalid1": "dddd",
-          "SomethingInvalid2": "eeee"
+          "SomethingInvalid2": "eeee",
+          "Credentials": {
+              "RoleArn": "arn:aws:iam::111122223333:role/LambdaRole"
+	  }
```

Comment:
Not the best test, but confirms that `"Credentials"` doesn't raise a validation error in the `Additional properties that are not valid` suite

Reply:
Alignment of the trailing `}` looks funny - is there a tab in there?

Reply:
:+1: Good catch I don't have `.ts` files handled in my vimrc for tabs-as-spaces